.github/workflows/validate-and-test.yml

```diff
@@ -0,0 +1,73 @@
+name: Validate and test
+
+on:
+  pull_request:
+  push:
+    branches: [ master ]
+    tags-ignore: [ v* ]
+
+jobs:
+  formatting-check:
+    name: Check for missing formatting
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up JDK ${{ matrix.JABBA_JDK }}
```

Comment:
`matrix.JABBA_JDK` won't be available here.

Reply:
```suggestion
      - name: Set up JDK 8
```

Reply:
Great, thanks, these were left-over from a previous version...